src/script/api/squirrel_export.sh

```diff
@@ -43,57 +43,34 @@ case $apilc in
 	*) echo "Unknown API type."; exit 1 ;;
 esac
 
-if [ -z "$1" ]; then
-	for f in `ls ../*.hpp`; do
-		bf=`basename ${f} | sed s@script_@${apilc}_@`
-
-		# ScriptController has custom code, and should not be generated
-		if [ "`basename ${f}`" = "script_controller.hpp" ]; then continue; fi
-		if [ "`basename ${f}`" = "script_object.hpp" ]; then continue; fi
-
-		${AWK} -v api=${apiuc} -f ${scriptdir}/squirrel_export.awk ${f} > ${bf}.tmp
-
-		if [ "`wc -l ${bf}.tmp | cut -d\  -f1`" = "0" ]; then
-			if [ -f "${bf}.sq" ]; then
-				echo "Deleted: ${bf}.sq"
-				svn del --force ${bf}.sq > /dev/null 2>&1
-			fi
-			rm -f ${bf}.tmp
-		elif ! [ -f "${bf}.sq" ] || [ -n "`diff -I '$Id' ${bf}.tmp ${bf}.sq 2> /dev/null || echo boo`" ]; then
-			mv ${bf}.tmp ${bf}.sq
-			echo "Updated: ${bf}.sq"
-			svn add ${bf}.sq > /dev/null 2>&1
-			svn propset svn:eol-style native ${bf}.sq > /dev/null 2>&1
-			svn propset svn:keywords Id ${bf}.sq > /dev/null 2>&1
-		else
-			rm -f ${bf}.tmp
-		fi
-	done
-else
-	${AWK} -v api=${apiuc} -f ${scriptdir}/squirrel_export.awk $1 > $1.tmp
-	if [ `wc -l $1.tmp | cut -d\  -f1` -eq "0" ]; then
-		if [ -f "$1.sq" ]; then
-			echo "Deleted: $1.sq"
-			svn del --force $1.sq > /dev/null 2>&1
+for f in `ls ../*.hpp`; do
+	bf=`basename ${f} | sed s@script_@${apilc}_@`
+
+	# ScriptController has custom code, and should not be generated
+	if [ "`basename ${f}`" = "script_controller.hpp" ]; then continue; fi
```

Comment:
The fact that this no longer excludes script_object.hpp doesn't change anything?

Reply:
script_object.hpp has @api none so it's excluded by itself